sys/arm64/arm64/locore.S

```diff
@@ -319,6 +319,15 @@ drop_to_el1:
 	/* Don't trap to EL2 for exceptions */
 	mov	x2, #CPTR_RES1
 	msr	cptr_el2, x2
+#if __has_feature(capabilities)
+	/*
+	 * Wait for the write to cptr_el2 to complete. It will enable the
```

Comment:
That's not quite what this does; any instruction that doesn't rely on previous MSR changes are still allowed to execute. I'd simplify this comment a lot to maybe something like:

> For Morello we need a context synchronisation event as we use capabilities before the later eret.